MOBILE_API.xml

```diff
@@ -395,7 +409,7 @@
     </enum>
     
     <enum name="HmiZoneCapabilities">
-        <description>Contains information about the HMI zone capabilities. (For future use.)</description>
+        <description>Contains information about the HMI zone capabilities.</description>
```

Comment:
I don't think we should remove the future use. I'm fairly certain this isn't used at all and needs to be deprecated.